src/main/webapp/jsp/module/ar/FederalFinancialReport.jsp

```diff
@@ -0,0 +1,116 @@
+<%--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2018 Kuali, Inc.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+--%>
+<%@ include file="/jsp/sys/kfsTldHeader.jsp" %>
+
+<c:set var="award" value="${DataDictionary.CuAward.attributes}"/>
+<c:set var="agency" value="${DataDictionary.Agency.attributes}"/>
+<c:set var="organization" value="${DataDictionary.OrganizationAccountingDefault.attributes}"/>
+
+<kul:page showDocumentInfo="false"
+          headerTitle="Federal Financial Report Generation"
+          docTitle="Federal Financial Report Generation" renderMultipart="true"
+          transactionalDocument="false" htmlFormAction="arFederalFinancialReport"
+          errorKey="foo">
+
+    <div id="lookup">
+        <div class="main-panel">
+            <div class="headerarea-small"></div>
+            <table class="standard" style="margin: 20px auto 0 auto;" summary="Federal Financial Report Generation"
+            ">
```

Comment:
This quotation mark doesn't have a corresponding match. The base file from KualiCo already has this problem, so I'm fine with moving this forward as-is and then we can create a ticket to have KualiCo fix it.